elide-integration-tests/src/test/java/com/yahoo/elide/tests/ResourceIT.java

```diff
@@ -1632,6 +1632,23 @@ public void testPostInvalidRelationship() {
                 .statusCode(HttpStatus.SC_BAD_REQUEST);
     }
 
+    @Test(priority = 42)
+    public void testSortByIdTopLevel() {
+        String sortByIdAscendingTopLevel = jsonParser.getJson("/ResourceIT/sortByTopLevelAscending.json");
+        given()
+                .accept(JSONAPI_CONTENT_TYPE)
+                .get("/parent?sort=id")
```

Comment:
Will add a test for non-rootable collections too but I suggest we punt on the total IT refactor (at least in this PR). That way we don't hold up this feature for too long.